app/Models/Patient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Patient extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'patient_name',
+        'phone_number',
+        'email',
+    ];
+
+
+//    public function user():BelongsTo
+//    {
+//        return $this->belongsTo(User::class);
+//    }
+
+    public function user(): BelongsToMany
```

Comment:
When you refer to multiple instances, you should name the function in the plural form.